### PR TITLE
freeplane: 1.9.14 -> 1.11.8

### PR DESCRIPTION
**Pull request by maxhbr**
## Description of changes

This builds on https://github.com/NixOS/nixpkgs/pull/244937 and adds another bump from 1.11.4 to 1.11.8 on top.

~~This PR still has open issues.~~ All open issues are resolved

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) (or backporting [23.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md) and [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

<details>
  <summary>outdated issue with this PR</summary>

## ~~Open issues~~ fived via https://github.com/NixOS/nixpkgs/pull/275479/commits/b444ec34ebcfe2ca00b825c81afeac4375adf855#diff-39091065f4ad22ac853cb58126f4841171696ed247b18e60a42d19ba4e0e0829R59-R60
it currently does not compile and the gradle build fails with:
```
FAILURE: Build failed with an exception.                                                                                                           
                                                                                                                                                   
* Where:                                                                                                                                           
Build file '/build/source/build.gradle' line: 186                                                                                                  
                                                                                                                                                   
* What went wrong:                                                                                                                                 
A problem occurred configuring project ':freeplane_plugin_script'.                                                                                 
> Could not resolve all files for configuration ':freeplane_plugin_script:lib'.                                                                    
   > Could not resolve info.picocli:picocli:4.7.4.                       
     Required by:                                                        
         project :freeplane_plugin_script > org.apache.groovy:groovy-all:4.0.13 > org.apache.groovy:groovy-cli-picocli:4.0.13
      > No cached version of info.picocli:picocli:4.7.4 available for offline mode.                                                                
      > No cached version of info.picocli:picocli:4.7.4 available for offline mode.                                                                
      > No cached version of info.picocli:picocli:4.7.4 available for offline mode.                                                                
      > No cached version of info.picocli:picocli:4.7.4 available for offline mode.
```
the dependency is in a weird path in the deps derivation:
```
> find /nix/store/zgkwpv58w4jr7r3zgvr3sl8sf280jkpa-freeplane-deps-1.11.8/ -iname '*picocli*'
/nix/store/zgkwpv58w4jr7r3zgvr3sl8sf280jkpa-freeplane-deps-1.11.8/org/apache/groovy/groovy-cli-picocli
/nix/store/zgkwpv58w4jr7r3zgvr3sl8sf280jkpa-freeplane-deps-1.11.8/org/apache/groovy/groovy-cli-picocli/4.0.13/groovy-cli-picocli-4.0.13.jar
/nix/store/zgkwpv58w4jr7r3zgvr3sl8sf280jkpa-freeplane-deps-1.11.8/org/apache/groovy/groovy-cli-picocli/4.0.13/groovy-cli-picocli-4.0.13.pom
/nix/store/zgkwpv58w4jr7r3zgvr3sl8sf280jkpa-freeplane-deps-1.11.8/share/info/picocli
/nix/store/zgkwpv58w4jr7r3zgvr3sl8sf280jkpa-freeplane-deps-1.11.8/share/info/picocli/picocli
/nix/store/zgkwpv58w4jr7r3zgvr3sl8sf280jkpa-freeplane-deps-1.11.8/share/info/picocli/picocli/4.7.4/picocli-4.7.4.jar
/nix/store/zgkwpv58w4jr7r3zgvr3sl8sf280jkpa-freeplane-deps-1.11.8/share/info/picocli/picocli/4.7.4/picocli-4.7.4.pom
```
the path instead of `.../share/info/picocli/picocli/...` should be `.../info/picocli/picocli/...` and a `mv "$out/share/info" "$out"` fixes the build.

<details>

### CODE REVIEW

**Comment by maxhbr** [comment]
Is there some magic after install that moves `$out/info` to `$out/share/info`?

File: pkgs/applications/misc/freeplane/default.nix [hunk 1]
```diff
@@ -43,80 +66,88 @@ let
...
     runHook postInstall
   '';
 
   meta = with lib; {
     description = "Mind-mapping software";
     homepage = "https://freeplane.org/";
     license = licenses.gpl2Plus;
-    platforms = platforms.linux;
-    maintainers = with maintainers; [ chaduffy ];
+    # platforms = platforms.linux;
```
Comment (SuperSandro2000) [inline pkgs/applications/misc/freeplane/default.nix:148]:
```suggestion
    platforms = platforms.linux;
```
we should probably keep this

File: pkgs/applications/misc/freeplane/default.nix [hunk 2]
```diff
@@ -43,80 +66,88 @@ let
...
+    maintainers = with maintainers; [
+      chaduffy
+    ];
```
Comment (SuperSandro2000) [inline pkgs/applications/misc/freeplane/default.nix:151]: Not sure why this line was changed

File: pkgs/applications/misc/freeplane/default.nix [hunk 3]
```diff
@@ -43,80 +66,88 @@ let
...
+    jdk
+    gradle
+  ];
 
   buildPhase = ''
-    mkdir -p -- ./freeplane/build/emoji/{txt,resources/images}
-    cp ${emoji}/emoji/txt/emojilist.txt ./freeplane/build/emoji/txt/emojilist.txt
-    cp -r ${emoji}/resources/images/emoji ./freeplane/build/emoji/resources/images/emoji
-    GRADLE_USER_HOME=$PWD gradle -Dorg.gradle.java.home=${jdk} --no-daemon --offline --init-script ${gradleInit} -x test -x :freeplane:downloadEmoji build
+    mkdir -p -- ./freeplane/build
```
Comment (SuperSandro2000) [inline pkgs/applications/misc/freeplane/default.nix:97]:
```suggestion
    mkdir -p freeplane/build
```
nit

**Review by SuperSandro2000 — APPROVED** [review]

File: pkgs/applications/misc/freeplane/default.nix (hunk 2 above)
Comment (maxhbr) [reply pkgs/applications/misc/freeplane/default.nix:151]: seems to be some auto formatting. Was already in the commit by @charles-dyfis-net

**Comment by maxhbr** [comment]
Hey @SuperSandro2000 , thanks for your review. I applied your suggestions.

**Comment by pbsds** [comment]
Result of `nixpkgs-review pr 275479` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>freeplane</li>
  </ul>
</details>

**Pull request merged**